source/ddbus/conv.d

```diff
@@ -1,9 +1,11 @@
 module ddbus.conv;
 
+import ddbus.attributes : isAllowedField;
```

Comment:
`ddbus.conv` should publically import `ddbus.attributes` in case someone imported it to get these UDAs (`public import`)

Reply:
not really, because the UDAs are not yet in any release. so it can still be changed without breaking anything outside this repo.

Reply:
Hm I guess you're right but don't forget to include it in package.d then so it comes with `import ddbus`